addons/website/tests/test_website_visitor.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+from odoo.tests import HttpCase
+
+
+class WebsiteVisitorTests(HttpCase):
+    def test_create_visitor_on_tracked_page(self):
+        Page = self.env['website.page']
+        View = self.env['ir.ui.view']
+        Visitor = self.env['website.visitor']
+        base_view = View.create({
+            'name': 'Base',
+            'type': 'qweb',
+            'arch': '''<t name="Homepage" t-name="website.base_view">
+                        <t t-call="website.layout">
+                            I am a generic page
+                        </t>
+                    </t>''',
+            'key': 'test.base_view',
+        })
+        untracked_page = Page.create({
```

Comment:
If the model supports create_multi, you could use that to save one request (every bit counts!):

```
pages = Page.create([{
    'view_id': base_view.id,
    'url': '/untracked_page_1',
    'website_published': True,
}, {
    'view_id': base_view.id,
     'url': '/tracked_page_1',
     'website_published': True,
     'is_tracked': True,
}])
untracked_page = pages[0]
tracked_page = pages[1]
```